packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -57,5 +57,6 @@
   "force_source_delete_this_source_text": "SFDX: Delete This from Project and Org",
   "force_config_set_org_text": "SFDX: Set a Default Org",
   "internal_dev_description": "Specifies whether or not to set up internal development mode which provides limited SFDX commands.",
-  "refresh_org_browser_text": "Refresh Org Browser"
+  "refresh_org_browser_text": "Refresh Org Browser",
+  "force_source_retrieve_display_text": "Retrieve Source from Org"
```

Comment:
@hbc-bindu The hover text would look like this: 
![Screen Shot 2019-07-15 at 3 14 18 PM](https://user-images.githubusercontent.com/21326913/61252805-a4a4ed00-a713-11e9-85ce-8a82d45fbdcc.png)

Let me know if that looks okay!